cci_rth.py

```diff
@@ -2,7 +2,10 @@
 
```

Comment:
I'm not certain that this module, or the pyinstaller dep in general are still in use...

Reply:
I guess not, given that the import here was out of date! I'll remove the files related to pyinstaller.

Reply:
you rock